packages/fontforge-of-ocaml/fontforge-of-ocaml.1.1.0/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
```

Comment:
We try to at least get `opam-version:` be the first field. I’m not sure how your opam file got several others before

Reply:
That file was generated automatically by `dune` from a template. In that template, `opam-version: "2.0"` is the first line. 
In fact, `dune` generated automatically all the lines before from the `dune-project` file.

Reply:
Why do you need this template file? As far as I can see everything can be written in the dune-project file (or even better imo, autogeneration is not needed)